### ISSUE TITLE
Rclone bisync erroring out with `–links`

### ISSUE THREAD

**Issue by callegar** [rclone/rclone#6572]
#### The associated forum post URL from `https://forum.rclone.org`

https://forum.rclone.org/t/rclone-bisync-erroring-out-with-links/33877

#### What is the problem you are having with rclone?

Using bisync together with --links can fail. Reproducible via attached script.

#### What is your rclone version (output from `rclone version`)

rclone v1.60.0
- os/version: arch 22.0.0 (64 bit)
- os/kernel: 6.0.8-1-MANJARO (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.19.2
- go/linking: dynamic
- go/tags: none

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

Linux (Manjaro) 64 bit

#### Which cloud storage system are you using? (e.g. Google Drive)

OneDrive (business)

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

`rclone bisync TestOD OneDrive:TestOD`

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

Test script by which the log was obtained

```sh
#!/bin/sh

set -x

mkdir TestOD
cd TestOD
touch foo
ln -s foo bar
cd -
rclone -l sync TestOD OneDrive:TestOD
rclone -l bisync -1 TestOD OneDrive:TestOD
cd TestOD
mkdir NewPlace
mv foo bar NewPlace
cd -
rclone -vv -l bisync --force TestOD OneDrive:TestOD
```

The log below is obtained from the full test script, so that all the operations are logged. Only the final call to `rclone` (the one triggering the error) is made verbose.

[log-x.txt](https://github.com/rclone/rclone/files/10022891/log-x.txt)

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by callegar** [comment]
Issue also present with v1.60.1

**Comment by ncw** [comment]
Good repro @callegar . Have you got time to investigate the code? Unfortunately our `bisync` developer isn't available at the moment.

**Comment by callegar** [comment]
> Have you got time to investigate the code?

Not really... is there some expectation about when the bisync main developer might be able to take a look at it? Unfortunately, I do not know the code ...and it is go, that I have never used before...

**Comment by callegar** [comment]
I see that there is new activity on bisync from a pull request dated 23 Apr by @nielhash. I wonder if this might be a chance to have someone expert in bisync look at this issue...

**Comment by nielash** [comment]
Thanks for the heads up, @callegar! I looked into this a bit tonight, and I can reproduce it in the latest stable version ([v1.62.2](https://github.com/rclone/rclone/releases/tag/v1.62.2)), but I canNOT reproduce it in [the latest beta](https://beta.rclone.org/). So it appears to me that the underlying issue has been fixed recently (and not by anything I did, since that's not yet merged.) At a quick glance, I'd guess [64cf9ac](https://github.com/rclone/rclone/commit/64cf9ac911529bad9b62b09a4012e84a1e406c6f) might be the commit responsible, but I haven't tested that hypothesis and it might well be wrong.

Are you able to try the beta and see if it resolves the issue for you as well? If you find that it does not, please feel free to let me know and I'd be happy to look into it further.

Also, thank you for the excellent test script -- that made testing much easier 😃

**Comment by nielash** [comment]
I'm closing this, as I believe the issue was fixed in [v1.63.0](https://github.com/rclone/rclone/releases/tag/v1.63.0) and there's been no response suggesting otherwise. If anyone disagrees, please feel free to reply / reopen. 🙂
